src/app/plugins/plugin_colorcalib.cpp

```diff
@@ -140,3 +153,67 @@ void PluginColorCalibration::mouseReleaseEvent ( QMouseEvent * event, pixelloc l
 void PluginColorCalibration::mouseMoveEvent ( QMouseEvent * event, pixelloc loc ) {
   mouseEvent(event,loc);
 }
+
+void PluginColorCalibration::slotCopyLUT() {
+    copyLUT();            //perform memory copy of other camera LUT
+}
+
+void PluginColorCalibration::copyLUT() {            //perform memory copy of other camera LUT
+    //TODO: replace with a method that can fetch known colors
+    // use: v_lut_colors->addItem("all");
+    if (v_lut_sources->getCount()==1 && v_lut_sources->getString()==LUT_COPY_PLACEHOLDER) {
+        VarString *pSelf = reinterpret_cast<VarString*>(settings->getParent());
+        std::vector<VarType *> vectRelatives = settings->findRelatives("LUT 3D", true);  //find LUT3D nodes node
+        if (vectRelatives.size() < 1) {     //nothing like LUTs? abort
+            fprintf(stderr, "LUT3D: Could not find any LUT3D nodes for sources, aborting copy.\n");
+        }
+        else {
+            v_lut_sources->setSize(0);          //truncate/clear former list
+            for (int iCam=0; iCam<vectRelatives.size(); iCam++) {
+                VarString *pCamera = reinterpret_cast<VarString*>(vectRelatives[iCam]->getParent()->getParent());
+                if (pCamera != pSelf)
+                    v_lut_sources->addItem(pCamera->getName());
+            }
+            if (!vectRelatives.empty())
+                v_lut_sources->selectIndex(0);
+            v_lut_colors->setSize(0);          //truncate/clear former list
+            v_lut_colors->addItem("all");
+            v_lut_colors->selectIndex(0);
+        }
+    }
+    else {              //user has selected a camera
+        bool bSuccess = true;
+        VarBlob *pSource = NULL;
+        int color_index = -1;       //default -1 is ALL colors; TODO be from var 'v_lut_colors'
+        std::vector<VarType *> vectTarget = settings->findRelatives(v_lut_sources->getString(), true);
+        if (vectTarget.size()!=1) {
+            fprintf(stderr, "LUT3D: Found too many cameara nodes (%d) for camera '%s', aborting copy.\n",
+                    static_cast<int>(vectTarget.size()), v_lut_sources->getString().c_str());
+            bSuccess = false;
+        }
+        else {
+            vectTarget = vectTarget[0]->findRelatives("LUT Data", false);  //just children
+        }
+        if (bSuccess && vectTarget.size()!=1) {
+            fprintf(stderr, "LUT3D: Found too many LUT nodes (%d) for camera '%s', aborting copy.\n",
+                    static_cast<int>(vectTarget.size()), v_lut_sources->getString().c_str());
+            bSuccess = false;
+        }
+        else {
+            pSource = reinterpret_cast<VarBlob*>(vectTarget[0]);
+            
+            //WARNING: Makes assumption that all LUT blobs have the same properties
```

Comment:
Unfortunately, the way the VarTypes are used to deeply copy and sequester data, there is no great way to propagate a modification through the rest of the application (through to processing, data storage, LUT).  Admittedly the copy is dangerous, but it is size-checked in the LUT class itself and there aren't any other great ways that I could find in the near-term.